src/platform_impl/linux/wayland/seat/pointer/mod.rs

```diff
@@ -109,9 +109,10 @@ impl WinitPointer {
         let serial = Some(self.latest_serial.get());
         for cursor in cursors {
             if self.pointer.set_cursor(cursor, serial).is_ok() {
-                break;
+                return;
             }
         }
+        error!("Failed to set cursor");
```

Comment:
Minor nit: This should maybe be a warning instead, since the system can easily continue without the cursor having been set?

Reply:
I have changed the `error!()` to a `warn!()`. This does make it a bit inconsistent with the same error as it is logged by `smithay_client_toolkit::window::concept_frame`, but that will likely be a non-issue if #1990 is merged (as the concept frame will be gone by then).